packages/typespec-azure-resource-manager/test/rules/arm-resource-provisioning-state.test.ts

```diff
@@ -60,7 +83,7 @@ describe("typespec-azure-resource-manager: arm resource provisioning state rule"
       .toEmitDiagnostics({
         code: "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state",
         message:
-          "The RP-specific property model in the 'properties' property of this resource must contain a 'provisioningState property.  The property type should be an enum, and it must specify known state values 'Succeeded', 'Failed', and 'Canceled'.",
+          "The RP-specific property model in the 'properties' property of this resource must contain a 'provisioningState property.  The property type should be an enum or a union of string values, and it must specify known state values 'Succeeded', 'Failed', and 'Canceled'.",
```

Comment:
Do we still need to mention `enum` here?

Reply:
for now we do as this is a hotfix where we just allow both state to live side by side. We have an issue to do the cleanup for this sprint that will be in next release.